deploy/olm-catalog/3scale-operator/0.0.1/apps_v1alpha1_apimanager_crd.yaml

```diff
@@ -0,0 +1 @@
+../../../crds/apps_v1alpha1_apimanager_crd.yaml
```

Comment:
The CRD files needs to be in the same _bundle_ directory, so I am placing there as symbolic links to avoid maintaining duplicates.

Reply:
But if source CRD's change, then you will have 0.0.1 version pointing to incompatible CRD's. 

Maintaining duplicates seems fair trade off.

Reply:
Agree that something was wrong with this strategy, but if the master branch always maintains the latest CRD, why should it have a directory with a specific version number? I suggest the following:

deploy
>> olm-catalog
>>>> community (in the future / probably not needed yet)
>>>> redhat

In the redhat directory, there could be symlinks to the latest CRDs (for now, just one CRD). If you want to deploy a specific version to olm, you use branch/commit. WDYT?

Reply:
yes! ... and no :) 

we would like to go a step further and have a single directory. Then differences for community and redhat bundles in different branches.

Reply:
I will change the PR #71 on that direction